fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt

```diff
@@ -98,7 +98,13 @@ object SupportUtils {
         return "https://www.mozilla.org/$langTag/$path"
     }
 
-    fun getWhatsNewUrl(context: Context) = getSumoURLForTopic(context, SumoTopic.WHATS_NEW)
```

Comment:
Is `SumoTopic.WHATS_NEW` used anywhere or is the URL pointed by it relevant anymore after this change? Could we remove it if it isn't relevant anymore?

Reply:
good catch, I've removed the unused enum and cleaned up the code a bit.